config/routes.rb

```diff
@@ -1,7 +1,39 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: "sales#index"
-  get 'sales/index'
+  resources :campaigns
+
+  root to: 'sales#index'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
 end
+
+
+
+
+=begin
```

Comment:
why is this here?

Reply:
i was commenting out a piece of code

Reply:
it has nothing to do with routes. Delete it. Whether it's a comment or not, things should be where they should be. Use your IDEs and notes for comments not related to the code